save-common/src/jvmMain/kotlin/org/cqfn/save/entities/AgentStatus.kt

```diff
@@ -0,0 +1,14 @@
+package org.cqfn.save.entities
+
+import org.cqfn.save.agent.AgentState
+import java.sql.Date
+import javax.persistence.*
+
+@Entity
+class AgentStatus (
```

Comment:
There should also be a field for agent id or container id, not only generated id for DB

Reply:
Fixed